acceptance-tests/src/test/resources/features/site/footer.feature

```diff
@@ -6,41 +6,63 @@ Feature: Footer
 
     Scenario: Terms and conditions
         Given I navigate to the "home" page
-        When I can click on the "Terms and Conditions" link
+        When I can click on the "Terms and conditions" link
         Then I should see a page titled "Terms and Conditions"
         And I should see the content "1              About these terms ..."
 
+    Scenario: Accessibility help
+        Given I navigate to the "home" page
+        When I can click on the "Accessibility help" link
+        Then I should see a page titled "Accessibility help"
+        And I should see the content "Our site aims to comply with the World Wide Web Consortium's (W3C's) Web Accessibility Guidelines ..."
+
 
-    # Tests for each of the pages on the site
+    # Tests for each of the pages on the site can see footer, and a footer link works
     Scenario: Publications Overview
         Given I navigate to the "publications overview" page
         Then I should see the footer
+        When I can click on the "Terms and conditions" link
```

Comment:
It will probably read better on these if you get rid of the 'can'

Reply:
Yep, "can" now removed.